### DISCUSSION TITLE
Scheduling Automatic backup of longhorn volume on amazon s3 storage

### DISCUSSION THREAD

**Discussion by ravisingh-adeptia** [longhorn/longhorn#5967] [Q&A]
Can longhorn automatically schedule taking backup of a selected or all volumes on amazon s3 storage at certain interval.

**Comment by ChanYiLin** [comment]
You can refer to this doc: https://longhorn.io/docs/1.4.2/snapshots-and-backups/scheduling-backups-and-snapshots/
